src/apps/bridge/sm_config_crc_list.cpp

```diff
@@ -69,6 +69,20 @@ void SMConfigCRCList::clear() {
   encode();
 }
 
+/*!
+  \brief Get the list of CRCs.
+
+  This function reads the list from the configuration.
+
+  \param[out] num_crcs[out] The number of CRCs in the list.
+  \return A pointer to the list of CRCs.
+*/
+uint32_t *SMConfigCRCList::get(uint32_t &num_crcs) {
+  decode();
+  num_crcs = _num_crcs;
+  return _crc_list;
```

Comment:
So this makes me nervous because it provides a public API to get a pointer to a private member. Before this change there was no danger of an external user of this object modifying or corrupting the object itself, but this function opens the door for such mistakes in the future. This PR uses the returned pointer safely, but future folks may not be so careful.

I love the idea of logging the full list; that's very helpful for debugging.

Alternatives I can see easily are (1) allocating a copy of the list and requiring the caller to free, or (2) building the printed string internally and returning that, requiring the caller to free. Thoughts?